app.py

```diff
@@ -99,9 +99,11 @@ def pem_create():
     output_stream = StringIO
 
     if request.method == 'POST':
-        name = request.form['pem-name']
+        name = request.form['name']
         try:
             key_pair, key_finger_print = create_pem(name)
+            outfile = open('./.pem/' + name + '.pem','w')
```

Comment:
Writing mode should be 'w+' not 'w'.
If you tried to overwrite contents, 'w' can not handle it.